examples/services/vars.tf

```diff
@@ -0,0 +1,43 @@
+# Kubernetes admin credentials
+variable "admin_password" { default="oa2zao0yeezau0xaeYe3oedu3aiphu" }
```

Comment:
I don't think a default should be provided for this. It could be easily used and not replaced